grease-cli/tests/llvm/declare-in-override/declare-in-override.config

```diff
@@ -1 +0,0 @@
---overrides tests/llvm/declare-in-override/my-malloc.llvm.cbl
```

Comment:
Why was this file moved?

Reply:
The `.config` file moved to be next to the `.llvm.cbl` file, which also moved. The `.llvm.cbl` file moved up a directory so that it was next to the other `.llvm.cbl` files, and so could be discovered by the `Dir.listDirectory` in the harness. Similar treatment was applied to the test in `startup-override/`.